src/index.ts

```diff
@@ -90,174 +70,23 @@ async function installTypeScriptAsNeeded(tsLocal: string | undefined, onlyTestTs
 }
 
 function usage(): void {
-    console.error("Usage: dtslint [--version] [--installAll] [--onlyTestTsNext] [--expectOnly] [--localTs path]");
+    console.error("Usage: dtslint [--version] [--installAll] [--onlyTestTsNext] [--localTs path]");
     console.error("Args:");
     console.error("  --version        Print version and exit.");
     console.error("  --installAll     Cleans and installs all TypeScript versions.");
-    console.error("  --expectOnly     Run only the ExpectType lint rule.");
```

Comment:
cc @43081j - out of curiosity, did you end up looking at this? I'm a little irked at how complex this is in the scenario I outlined in the OP 😞

Reply:
we'd be programmatically calling eslint, right? wouldn't we be able to specify the rule set to run in that case, to just this rule? on the command line, you'd have to do it by a custom eslint or `--no-eslintrc` i think. but in code i'd assume you can pass the rule in by itself

Reply:
Yeah my sadness there is that if someone has a deeply nested set of directories, each with their own `.eslintrc.json`, we'd have to compute from each of those config files what the actual set of rules to run is. Each ESLint config could specify something different for an expect-type* rule.

Maybe I'm overthinking this?

Reply:
I think so, what I'm saying is we will be programatically calling it anyway. So just like in unit tests we could enable only one rule (expect type rule). Ignoring any user defined rule sets. I think it'd be part of dtslint rather than the plugin holding the rules